client/components/common/search-results.vue

```diff
@@ -84,6 +84,7 @@ export default {
     searchRestrictLocale: sync('site/searchRestrictLocale'),
     searchRestrictPath: sync('site/searchRestrictPath'),
     results() {
+      this.cursor = 0
```

Comment:
Don't modify the state from within a computed getter. Use a watcher instead, e.g.:

```js
watch: {
  results () {
    this.cursor = 0
  }
}
```